lib/basedriver/capabilities.js

```diff
@@ -20,16 +20,19 @@ function mergeCaps (primary = {}, secondary = {}) {
 }
 
 // Validates caps against a set of constraints
-function validateCaps (caps, constraints = {}, skipPresenceConstraint = false) {
+function validateCaps (caps, constraints = {}, opts = {}) {
```

Comment:
Can you please sync your changes with @KazuCocoa (https://github.com/appium/appium-base-driver/pull/164)

Reply:
Yeah, once we approve #164 I can take care of rebasing.

Reply:
I would ask you to take a look at that PR, so you know what is going to be changed

Reply:
Yeah I've looked at it and added some comments. Should be a pretty simple fix.